### ISSUE TITLE
Powerseal cannot work with OIDC authenticated kubernetes api server

### ISSUE THREAD

**Issue by iammuho** [bloomberg/powerfulseal#119]
I've configured my kubernetes to be authenticated by OIDC but now powefulseal gets an errror as `forbidden` 

nodes is forbidden: User \"system:anonymous\" cannot list resource \"nodes\" in API group \"\" at the cluster scope","reason":"Forbidden","details":{"kind":"nodes"},"code":403}

Anyone faced this issue before ? I can use my kubectl and my kubeconfig has been configured to pass over openid provider but powerfulseal cannot.

best,

**Comment by seeker89** [comment]
Hi, could you specify what version of Powerfulseal you are running ?

**Comment by geass** [comment]
Hello,

I've installed the powefulseal with `pip3 install powerfulseal` and my command is `powerfulseal -v interactive --aws --inventory-kubernetes`

powerfulseal version is 

`➜  powerfulseal git:(master) pip3 show powerfulseal
Name: powerfulseal
Version: 2.0.1
Summary: PowerfulSeal - a powerful testing tool for Kubernetes clusters
Home-page: https://github.com/bloomberg/powerfulseal
Author: Mikolaj Pawlikowski
Author-email: mikolaj@pawlikowski.pl
License:
                                 Apache License
                           Version 2.0, January 2004
                        http://www.apache.org/licenses/
Location: /usr/local/lib/python3.7/site-packages
Requires: ConfigArgParse, Flask, termcolor, openstacksdk, spur, kubernetes, PyYAML, jsonschema, boto3, future, requests, prometheus-client, flask-cors, flask-swagger-ui, coloredlogs, six
Required-by:`

**Comment by seeker89** [comment]
Hi, I might have some bandwidth to work on this in the near future. Could you point me to the relevant documentation to get me started ? Thanks.

**Comment by seeker89** [comment]
Duplicate of https://github.com/bloomberg/powerfulseal/issues/201
